index.html

```diff
@@ -177,4 +177,62 @@ <h4 class="g-item10">About Me</h4>
     </p>
   </section>
   <!--About Me section-->
+  <!--Form section-->
+  <section class="c-container" id="form-section">
+    <h5 class="h-form">CONTACT US</h5>
+    <P class="p-form">
+      If you have an application you are interested in developing, 
+      a feature that you need built or a project that needs coding.
+       I’d love to help with it 
+    </P>
+    <form action="https://formspree.io/f/mbjwogbd" method="post">
+      <ul class="non-style">
+      <li class="li-form">
+        <input 
+          class="text-field"
+          type="text" 
+          id="name" 
+          name="user_name"
+          placeholder="Text Input"
+          maxlength="30"
+          pattern="[a-zA-Z0-9]{1,30}"
+          required
+        >
+      </li>
+      <li class="li-form">
+        <input 
+          class="email-field"
+          type="email" 
+          id="mail" 
+          name="user_email"
+          placeholder="Dangelo@email.com"
+          required
+        >
+      </li>
+      <li class="li-form">
+        <textarea
+          class="msg-field"
+          id="msg" 
+          name="user_message"
+          placeholder="Enter text here"
+          required
+        ></textarea>
+      </li>
+      </ul>
+      <div class="centering-s">
+        <button class="g-item8 mr-s" type="submit">See Project</button>
```

Comment:
- [ ] Please implement the button hover and active state as shown in the Figma template 🙏